dlme_airflow/tasks/post_harvest.py

```diff
@@ -6,27 +8,27 @@
 from airflow.utils.task_group import TaskGroup
 
 from utils.catalog import catalog_for_provider
+from utils.ifpo_get_thumbnail_urls import add_thumbnail_urls  # The method name/include must match the value in metadata.post_harvest from the catalog
```

Comment:
As mentioned above, this include (i.e. `add_thumbnail_urls`) must match the value in metadata.post_harvest in the catalog.